src/functionalTests/java/uk/gov/hmcts/reform/finrem/functional/caseorchestration/AmendCaseDetailsTest.java

```diff
@@ -11,8 +11,8 @@
 import uk.gov.hmcts.reform.finrem.functional.IntegrationTestBase;
 
 import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertThat;
```

Comment:
org.junit.Assert.assertThat is now deprecated & should be replaced by hamcrest instance